WolvenKit.CR2W/Types/CR2WTypeManager.cs

```diff
@@ -114,13 +114,335 @@ public CR2WTypeManager()
+                    Register("W3MonsterHuntInvestigationArea", new CEntity(null)); //ws
+                    Register("W3SavedSoundClue", new CEntity(null)); //ws
+                    Register("W3SummonedFlies", new CEntity(null)); //ws
+                    Register("W3Poster", new CEntity(null)); //ws
+                        Register("W3LeaderboardCustom", new CEntity(null)); //ws
+                        Register("W3SavedPoster", new CEntity(null)); //ws
+                        Register("W3Signboard", new CEntity(null)); //ws
+                    Register("W3SavedAppearanceEntity", new CEntity(null)); //ws
+                    Register("W3FairytaleWitchFluid", new CEntity(null)); //ws
+                    Register("W3SpawnMarker", new CEntity(null)); //ws
+                    Register("W3StallEntity", new CEntity(null)); //ws
+                    Register("W3PointOfInterestEntity", new CEntity(null)); //ws
+                    Register("W3POIDispenser", new CEntity(null)); //ws
+                    Register("W3DestructSelfEntity", new CEntity(null)); //ws
+                    Register("W3NPCBackground", new CEntity(null)); //ws
+                    Register("W3NPCBackgroundPair", new CEntity(null)); //ws
+                    Register("W3NPCBackgroundWoman", new CEntity(null)); //ws
+                    Register("CBooksMinigameManager", new CEntity(null)); //ws
+                    Register("CBookMinigameSlot", new CEntity(null)); //ws
+                    Register("CBookMinigameBook", new CEntity(null)); //ws
+                    Register("CFactAdderOnCollisionWithTag", new CEntity(null)); //ws
+                    Register("W3CiriPhantom", new CEntity(null)); //ws
+                    Register("CTestTrigger", new CEntity(null)); //ws
+                    Register("W3KillTestTrigger", new CEntity(null)); //ws
+                    Register("W3BackgroundAnimatedEntity", new CEntity(null)); //ws
+                    Register("W3BackgroundAnimatedEntityTrigger", new CEntity(null)); //ws
+                    Register("W3ChangeCombatStageTrigger", new CEntity(null)); //ws
+                    Register("W3ShepherdGreetingTrigger", new CEntity(null)); //ws
+                    Register("W3DestructionTrigger", new CEntity(null)); //ws
+                    Register("W3FoodDispenser", new CEntity(null)); //ws
+            Register("W3HorseManager", new CEntity(null)); //ws
+
             // Testing area. Put your CVectors which don't work fully yet here.
 #if DEBUG
             // *.w2p
             //Register("CParticleEmitter", new CParticleEmitter(null));
             Register("CDecalSpawner", new CDecalSpawner(null));
             Register("SSeedKeyValue", new SSeedKeyValue(null));
-            Register("SDynamicDecalMaterialInfo", new SDynamicDecalMaterialInfo(null));
+            //Register("SDynamicDecalMaterialInfo", new SDynamicDecalMaterialInfo(null)); //rfuzzo: disabled because it couldn't read the stream
```

Comment:
These don't really need disabling. This section is for WIP vectors but not an error.